File: Java/Lista 3/L3_Ex1.4/src/UsuarioRegular.java

public class UsuarioRegular extends Usuario{
    
    public UsuarioRegular(String login) {
        super(login);
    }
    public void descricaoRegular(){
        System.out.println("O usuario regular pode ler arquivos e auterar arquivos");
    }
    
    public void dashboard(){
        System.out.println("Area de usuário cadastrado");
    }
    
    public void despedidaRegular(){
        System.out.println("Até logo" + this.getLogin() + "!");
    }
    
}

File: Swift/L3/Lista_03_EX04/UsuarioRegular.swift
import Foundation
 
 public class UsuarioRegular : Usuario{
 
	func descricaoRegular(){
		print("O usuario regular pode ler arquivos e auterar arquivos. ")
	}
	
	func dashBoard(){
		print("Area de usuário cadastrado")
	}
	
	func despedidaRegular( ) {
		print("Até logo \(login)!")
	}
	
}